src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/Asn1/X509ExtensionAsn.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Security.Cryptography.Asn1;
+using System.Security.Cryptography.X509Certificates;
+using Internal.Cryptography;
+
+namespace System.Security.Cryptography.X509Certificates.Asn1
+{
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct X509ExtensionAsn
```

Comment:
Missing source URL and ASN.1

Reply:
This was actually taken from the existing Pkcs code. I have another PR that moves the class over, so I will fix this after that one is merged and this one rebased.